manifests/plugin/ansible/runner.pp

```diff
@@ -0,0 +1,48 @@
+# = Ansible runner package installation
+#
+# $upstream_runner_repo:: If true, adds upstream repositories to install ansible-runner package from
+#
+class foreman_proxy::plugin::ansible::runner(
+  Boolean $upstream_runner_repo = $foreman_proxy::plugin::ansible::params::upstream_runner_repo,
```

Comment:
I think you can directly refer to the ansible class as well. Since you're passing it in ansible.pp, you can also leave out the default.

```suggestion
  Boolean $upstream_runner_repo = $foreman_proxy::plugin::ansible::upstream_runner_repo,
```

Reply:
I have found out, I do not need to pass the param out, if I am using it like you are suggesting. Is that correct way? Test are passing...